modules/installation-methods-for-different-architectures.adoc

```diff
@@ -14,10 +14,13 @@
 |*Supported architectures*|*Supported installation methods*
 
 |AMD64
-a|Operator (preferred), Helm charts, or `roxctl` CLI (not recommended)
+a|Operator (preferred), Helm charts, `roxctl` CLI (not recommended)
 
 | ppc64le ({ibm-power})
 .2+a|Operator
 
 | s390x ({ibm-z} and {ibm-linuxone})
+
+|aarch64 ({arm64})
+|Operator (preferred), Helm charts, `roxctl` CLI
```

Comment:
[The table immediately above this](https://84962--ocpdocs-pr.netlify.app/openshift-acs/latest/installing/acs-high-level-overview.html#install-platforms-methods_acs-high-level-overview) uses "or", and that matches standard grammar for a list format. 

```suggestion
a|Operator (preferred), Helm charts, or `roxctl` CLI (not recommended)

| ppc64le ({ibm-power})
.2+a|Operator

| s390x ({ibm-z} and {ibm-linuxone})

|aarch64 ({arm64})
|Operator (preferred), Helm charts, or `roxctl` CLI
```